src/main/java/net/logstash/logback/stacktrace/ShortenedThrowableConverter.java

```diff
@@ -150,7 +152,14 @@
      * True to print the root cause first.  False to print exceptions normally (root cause last).
      */
     private boolean rootCauseFirst;
-    
+
+    /**
+     * True to compute and inline stack hashes.
+     */
+    private boolean inlineHash;
```

Comment:
Should provide a way to set this to true via parsed options.  See the `parseOptions` method.